plugins/state_monitor.py

```diff
@@ -41,6 +44,10 @@ def __init__(
             If `method=='slack'`, then this is the channel in Slack to post notifications.
         slack_key_file : str or pathlib.Path, optional
             If `method=='slack'`, then the path to the file that contains the Slack key
+        google_chat_channel : str, optional
+            If `method=='google chat'`, then this is the channel in Google Chat to post notifications.
+        google_chat_key_file : str or pathlib.Path, optional
+            If `method=='google chat'`, then the path to the file that contains the Google Chat webhook key
```

Comment:
I can't quite reach the first line to make an actual suggestion, but these two would also be simplifed:

```rst
        channel : str, optional
            The chat channel in which to post notifications.
        key_file : str or pathlib.Path, optional
            The path to the file that contains the secret/webhook key for the given `method`.
```